python/MANIFEST.in

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
```

Comment:
oh yah not needed, probably from copying the license header for RAT from another Python file (and skipped since it starts with a `#`). removed :)